packages/p2p/src/components/order-details/__tests__/order-details-wrapper.spec.js

```diff
@@ -53,7 +53,7 @@ describe('<OrderDetailsWrapper />', () => {
         render(<OrderDetailsWrapper {...desktop_props} />);
 
         await waitFor(() => {
-            expect(screen.getByTestId('order-details-wrapper-desktop')).toBeInTheDocument();
+            expect(screen.getByTestId('dt_page-return')).toBeInTheDocument();
```

Comment:
```suggestion
            expect(screen.getByTestId('dt_page_return')).toBeInTheDocument();
```

The test ids name convention in our projects is this: `dt_lala_tada` :smile: